lib/takeaway.rb

```diff
@@ -0,0 +1,57 @@
+require 'menu'
+require 'order'
+require 'text'
+
+class Takeaway
+
+  attr_reader :menu, :order, :text
+
+  def initialize(menu = Menu.new, order = Order.new, text = Text.new)
```

Comment:
What is the responsibility of this class?
Although this provides a single interface for the user interaction, the class itself doesn't appear to have a responsibility - it only delegates. Be aware of patterns like this as it can make it harder to extend your code in some cases.